File: app/src/main/java/ua/te/hackathon/smartcity2015/ui/intro/IntroView.java
package ua.te.hackathon.smartcity2015.ui.intro;

import ua.te.hackathon.smartcity2015.ui.base.mvp.PresenterView;

/**
 * Created by nazarko on 2/13/16.
 */
public interface IntroView extends PresenterView {

  void showLoadingView();

  void hideLoadingView();

}

File: app/src/main/kotlin/ua/te/hackathon/smartcity2015/ui/intro/IntroView.kt
package ua.te.hackathon.smartcity2015.ui.intro

import ua.te.hackathon.smartcity2015.ui.base.mvp.PresenterView

/**
 * Created by nazarko on 2/13/16.
 */
interface IntroView : PresenterView {

  fun showLoadingView()

  fun hideLoadingView()

}
